be/src/storage/primary_key_encoder.cpp

```diff
@@ -244,22 +244,22 @@ bool PrimaryKeyEncoder::is_supported(const vectorized::Field& f) {
     }
 }
 
-bool PrimaryKeyEncoder::is_supported(const vectorized::Schema& schema) {
-    size_t n = schema.num_key_fields();
-    for (size_t i = 0; i < n; i++) {
-        if (!is_supported(*schema.field(i))) {
+bool PrimaryKeyEncoder::is_supported(const vectorized::Schema& schema, const std::vector<ColumnId>& sort_key_idxes) {
```

Comment:
For this class, `sort_key_idxes` naming is not ideal, change to `key_idxes`

Reply:
changed